src/python/pants/task/simple_codegen_task.py

```diff
@@ -293,6 +336,20 @@ def _inject_synthetic_target(
 
     return synthetic_target
 
+  def _original_export_specs(self, target):
+    collected_original_exports = getattr(target, 'export_specs', None)
```

Comment:
This should use the `def exports` property instead... `export_specs` is an implementation detail.